src/CookiePolicyBanner/index.jsx

```diff
@@ -35,22 +40,23 @@ class CookieBanner extends Component {
   }
 
   render() {
-    const langCode = getLanguageCode();
+    const ietfTag = getIETFTag();
```

Comment:
I think `lang` attribute values have to be `ISO-639-1`. The Spanish language cookie value (`es-419`) is not `ISO-639-1`. I could just take the first two characters of the parsed cookie value, but that doesn't seem as robust as mapping the cookie values (aka `IETF` values) to the appropriate `ISO-639-1` values. It could be argued that this is over-engineering, though.